core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-dashboard.js

```diff
@@ -15,26 +15,48 @@ function toggleCellDetails(detailsId) {
 
 function createStatusBlock() {
 
+  var totalCoresCount = $( "div#totalCores" ).data( "value" );
   var cpuUsage = $( "div#cpuUsage" ).data( "value" );
   var memoryUsage = $( "div#memoryUsage" ).data( "value" );
   // var heapUsageGauge = $( "div#heapUsage" ).data( "value" );
   // var offHeapUsageGauge = $( "div#offHeapUsage" ).data( "value" );
   var jvmHeapUsageGauge = $( "div#jvmHeapUsage" ).data( "value" );
 
+  var configCores = liquidFillGaugeDefaultSettings();
+  configCores.circleThickness = 0.15;
+  configCores.circleColor = "#3EC0FF";
+  configCores.textColor = "#3EC0FF";
+  configCores.textVertPosition = 0.5;
+  configCores.textSize = 1.5;
+  configCores.waveTextColor = "#00B0FF";
+  configCores.waveColor = "#A0DFFF";
+  configCores.waveAnimateTime = 1000;
+  configCores.waveAnimate = false;
+  configCores.waveRise = false;
+  configCores.waveHeight = 0;
+  configCores.waveHeightScaling = false;
+  configCores.waveOffset = 0.25;
+  configCores.waveCount = 2;
+  configCores.displayPercent = false;
+  configCores.minValue = 0;
+  configCores.maxValue = (totalCoresCount * 1.3);
+
+  var totalCoresCountGauge = loadLiquidFillGauge("totalCoresGauge", totalCoresCount, configCores);
+
```

Comment:
I assume this is the liquid fill display like used for memory etc now. There must be two counts to show relative measure, so that are the two counts here?

Reply:
Yes. This is liquid fill display. 
I did not understand second part in comment.

Reply:
Here we are trying to show total core count right? Liquid fill is for cases like "how much memory is used against total memory available" i.e. a relative percentage measure. What is totalCoreCount relative to here? Seems some arbitrary 1.3 factor.